### DISCUSSION TITLE
Overwriting Environment Variables

### DISCUSSION THREAD

**Discussion by Adam-Mould** [saleor/saleor#8666] [Q&A]
Hey there,

I'm running Saleor Core on an AWS EC2 instance. I have used Nginx and have set the configuration environment variables in a . bashrc file.

The initial set-up is working - I set my environment variables, cloned saleor core from Github, started the server etc.

The issue I'm now facing is I'm trying to change environment variables on an existing instance, but they don't seem to be taking effect in the saleor codebase.

For example, I decided to swap the database to use AWS RDS instead of being a local Postgres DB on the EC2 instance. So I carried out the following steps:
1. Changed the `DATABASE_URL` variable in _.bashrc_
2. Ran `source .bashrc`, so the variable change takes effect?
3. Restarted Nginx.

However, Saleor is still trying to connect to the old local Postgres DB. This also happens when I try and change `AWS_MEDIA_BUCKET_NAME` variable (to change the S3 bucket I'm using).

Is there a particular `python mange.py` command, or additional steps I need to run, in order for Saleor/Python/Django to accept the altered environment variables?

Thanks,
Adam

**Comment by arunodhyam** [comment]
Had a similar issue but it is a docker installation that got solved with rebuilding.
docker-compose down
docker-compose build
docker-compose up
